recipes/datashader/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "datashader" %}
+{% set version = "0.6.2" %}
+{% set sha256 = "fc76c791fc050b9564e6ecd57095ce42d1c10b11bbee751c796431b70daa85aa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bokeh/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - dask >=0.7.6
+    - datashape >=0.5.1
+    - numba >=0.35.0
+    - numpy >=1.7
+    - pandas >=0.15.0
+    - pillow >=3.1.1
+    - xarray >=0.9.6
+    - toolz >=0.7.4
+    - colorcet >=0.9.0
+    - param >=1.5.0,<2.0
+
+test:
+  requires:
+    - pytest >=2.8.5
+    - pytest-benchmark >=3.0.0
+    - rasterio
+    - scipy
+  imports:
+    - datashader
+
+about:
+  home: https://github.com/bokeh/datashader
+  license: New BSD
+  license_file: LICENSE.txt
+  summary: 'Data visualization toolchain based on aggregating into a grid'
+
+  description: |
+    Datashader is a data rasterization pipeline for automating the process of
+    creating meaningful representations of large amounts of data.
+  doc_url: https://datashader.readthedocs.io/
+  dev_url: https://github.com/bokeh/datashader
+
+extra:
+  recipe-maintainers:
+    - jbednar
```

Comment:
Also, just to confirm, happy to be a maintainer here, @jbednar?

Reply:
Yes.

Reply:
Thanks for confirming.